<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyBook</class>
 <widget class="QDialog" name="modifyBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>541</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>446</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>请输入需要修改的书的ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_id"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="ok_but">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>521</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,0,2">
    <item>
     <widget class="QComboBox" name="cbb_but">
      <item>
       <property name="text">
        <string>请选择修改内容</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>书名</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>作者</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>剩余数量</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>总数量</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line2"/>
    </item>
    <item>
     <widget class="QPushButton" name="change_but">
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
